justfile

```
@@ -13,4 +13,6 @@ docs-config := base-directory / "docs/mkdocs.yml"
 
 # Serve current docs located in ./docs/docs
 serve-docs port="8000":
+    typer {{base-directory}}/kpops/cli/main.py utils docs --name kpops --output {{base-directory}}/docs/docs/user/references/cli-commands.md
```

Shouldn't be like this?
```suggestion
    poetry run typer {{base-directory}}/kpops/cli/main.py utils docs --name kpops --output {{base-directory}}/docs/docs/user/references/cli-commands.md
```

---

It should